.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -1,97 +0,0 @@
-# This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yml and/or the recipe/meta.yaml.
-# -*- mode: yaml -*-
-
```

Comment:
Umm...looks like re-rendering removed the CI files here. Something is going wrong